Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -6493,6 +6504,10 @@ void HTMLMediaElement::playbackTargetPickerWasDismissed()
 
 void HTMLMediaElement::remoteHasAvailabilityCallbacksChanged()
 {
+#if ENABLE(MEDIA_SOURCE)
+    if (m_mediaSource)
+        m_mediaSource->openIfDeferredOpen();
+#endif
```

Comment:
this method is called whenever the Remote Playback API is disabled so I thought it would an easy path.
However, the JS binding simply set an attribute on the media element, to it's already covered by the `HTMLMediaElement::attributeChanged`